liota/dccs/iotcc.py

```diff
@@ -159,7 +146,8 @@ def on_response(msg):
             try:
                 log.debug("Received msg: {0}".format(msg))
                 json_msg = json.loads(msg)
-                log.debug("Processed msg: {0}".format(json_msg["type"]))
+                log.debug("Processing msg: {0}".format(json_msg["type"]))
+                self._check_version(json_msg)
```

Comment:
Redundant check, you are already raising the exception during the registration call if there is version mismatch the code will never reach here. It's an extra overhead to introduce version check in the current implemented model.